raster/src/main/scala/geotrellis/raster/io/geotiff/SegmentBytes.scala

```diff
@@ -16,17 +16,18 @@
 
 package geotrellis.raster.io.geotiff
 
-import spire.syntax.cfor._
-
 /**
- * The base trait of SegmentBytes. It can be implemented either as
- * an Array[Array[Byte]] or as a ByteBuffer that is lazily read in.
- */
-trait SegmentBytes extends Traversable[Array[Byte]] {
+  * Represents the bytes of the segments with-in a GeoTiff.
+  *
+  * The base trait of SegmentBytes. It can be implemented either as
+  * an Array[Array[Byte]] or as a ByteBuffer that is lazily read in.
+  */
+trait SegmentBytes extends Seq[Array[Byte]] {
```

Comment:
This had to changes from `Traversable` to `Seq` because the only way the former can tell its length is to iterate through the collection, which is obviously not great for streaming.